relay-server/src/utils/dynamic_sampling.rs

```diff
@@ -76,13 +76,8 @@ pub fn is_trace_fully_sampled(
         return Some(false);
     }
 
-    let adjustment_rate = match root_project_config.mode {
-        SamplingMode::Total => dsc.sample_rate,
-        _ => None,
-    };
-
     // TODO(tor): pass correct now timestamp
-    let evaluator = SamplingEvaluator::new(Utc::now()).adjust_client_sample_rate(adjustment_rate);
+    let evaluator = SamplingEvaluator::new(Utc::now());
 
```

Comment:
the TODO on top of this line, can it be removed? Or what does the TODO even mean?

Reply:
I noticed it as well, I will look into it.